src/column.rs

```diff
@@ -40,6 +39,13 @@ impl Statement<'_> {
         self.stmt.column_count()
     }
 
+    pub(crate) fn column_name(&self, col: usize) -> &str {
+        // Just panic if the bounds are wrong for now, we never call this
+        // without checking first.
+        let slice = self.stmt.column_name(col).expect("Column out of bounds");
```

Comment:
The `expect` is here instead of a panic inside the inner stmt's column_name mostly since I debated exposing this publicly, which would have had `column_name` return a `Option<&str>`. I can change that if you want.